src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx

```diff
@@ -139,7 +139,10 @@ export const SpaceEditFormNG: React.FC<SpaceEditFormNGProps> = ({
   const updateVenueRoom = useCallback(async () => {
     if (!user || !portalId) return;
 
-    const embedUrl = convertToEmbeddableUrl({ url: values.iframeUrl });
+    const embedUrl = convertToEmbeddableUrl({
```

Comment:
[NOTE]
`convertToEmbeddableUrl()` should probably be used inside `updateVenueNG()` to make sure every call to it has the URL updated based on `autoPlay` value that's been given to it, maybe even save raw and updated URL